hedera-node/hapi/src/testFixtures/java/module-info.java

```diff
@@ -1,7 +1,8 @@
-module com.hedera.node.hapi.fixtures {
+module com.hedera.node.hapi.test.fixtures {
```

Comment:
The plugin expects a clear relationship between a "Source Set" name and the Name of the Module in it. Now all test fixture modules have the `test.fixtures` suffix in their name (it was inconsistent before).